Server-side/src/projects/dto/create-project.dto.ts

```diff
@@ -11,9 +11,10 @@ export class CreateProjectDto {
   public title: String;
   @ApiProperty({
     description: 'language you want translate to',
-    example: 'ar',
+    example: ['ar', 'fr'],
   })
-  @IsNotEmpty()
-  @IsString()
-  targetLanguage: String;
+  @IsString({ each: true })
+  @IsArray()
+  @ArrayNotEmpty()
+  targetLanguage: [String];
```

Comment:
This will return an array of strings, 
for example: targetLanguage: ['مرحبا', 'Hello']
 The issue is how the frontend will distinguish between the Arabic ('ar') and English ('en') translations,
To resolve this, you should use an array of objects where each object has a key 'lang' and a value 'word'.